cli-docs/pow/pow_ffs_stage.md

```diff
@@ -1,10 +1,10 @@
 ## pow ffs stage
 
-Temporarily cache data in the Hot layer in preparation for pushing a cid storage config
+Temporarily stage data in the Hot layer in preparation for pushing a cid storage config
```

Comment:
Had the sensation that only using _stage_ will be better than introducing _cache_ which might have other semantic assumptions.